app/interface_student.py

```diff
@@ -0,0 +1,165 @@
+from aiogram import Bot, types
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data, clear_user_data
+from app.api.student import Student
+''', StudentJoinGroupException, StudentLeaveGroupException'''
+
+
+class StudentInterface:
+    def __init__(self, bot: Bot, student_name: str):
+        self.bot = bot
+        #self.database = Database() #добавить
+        self.student = Student(student_name)
+
+    async def process_invite_code(self, message: types.Message, invite_code: str):
+        try:
+            invite_code_int = int(invite_code)
+            student_tg_id = message.from_user.id
+            result = self.student.join_group(invite_code_int, student_tg_id)
```

Comment:
invite_code has type str in jon_group method https://github.com/six-nine/edu_queue/blob/main/app/api/student.py#L31

Reply:
fixed